apps/storybook/stories/config.ts

```diff
@@ -18,36 +18,19 @@ const demoIndexes: DemoIndexConfig = {
     placeholder: 'What do you want to learn about Orama?',
     sourceBaseUrl: 'https://docs.orama.com',
     sourcesMap: {
+      title: 'title',
       description: 'content',
     },
-    suggestions: [
-      'Why is Orama better than other search or AI solutions?',
-      'How does Orama ensure correct answers?',
-      'What are the steps to implement?',
-    ],
-    themeConfig: {
-      colors: {
-        light: {
-          '--text-color-primary': '',
-        },
-        dark: {
-          '--background-color-primary': '',
-          '--background-color-secondary': '',
-          '--background-color-tertiary': '',
-          '--border-color-primary': '',
-          '--backdrop-background-color-primary': '',
-        },
-      },
-    },
+    suggestions: ['What is Orama?', 'Does Orama have an integration with Strapi?', 'How to create an answer session?'],
     facetProperty: 'category',
     resultMap: {
+      title: 'title',
```

Comment:
@faustoq we expect to pass fields to `resultsMap` and `sourcesMap` only if they are different from 'title', 'description'. Did you pass 'title' here because things weren't working without it? Just trying to figure out if there was a regression here.

Reply:
I thought to make it explicit, to help developers looking at the stories as a reference / example of integration with the searchbox.